File: leetcode/1easy/위성원/2923.ts
function findChampion(grid: number[][]) {
  const go = (index: number) => {
    for (let i = 0; i < grid.length; i++) {
      if (grid[i][index] === 1) {
        return go(i)
      }
    }

    return index;
  }

  for (let i = 0; i < grid.length; i++) {
    for (let j = 0; j < grid[0].length; j++) {
      if (grid[i][j] === 1) {
        return go(i)
      }
    }
  }
};

File: leetcode/1easy/이경호/2923.cpp
class Solution {
public:
  int findChampion(vector<vector<int>>& grid) {
    for (int i = 0; i < grid.size(); i++) {
      int zc = 0;
      for (int j = 0; j < grid[i].size(); j++) {
        if (grid[i][j] == 0) {
          zc++;
        }
      }
      if (zc == 1) {
        return i;
      }
    }
    return -1;
  }
};